configs-testnet/entity.relayer.json

```
@@ -390,6 +390,53 @@
             }
         ]
     },
+    "BASE_GOERLI": {
+        "chain_name": "BASE_GOERLI",
+        "block_period_sec": 2,
+        "bootstrap_latest_height": 0,
```

모니터링을 시작해야하는 blockNumber 값을 주는 부분이라 contract들의 deploy_height 중 가장 낮은 값을 주면 됩니다.
```suggestion
        "bootstrap_latest_height": 8693872,
```